src/graphql/mutations/create_game_piece_actions.ts

```diff
@@ -113,16 +113,15 @@ async function handleRangedAttackAction(
   rangedAttackInput: Types.GamePieceRangedAttackActionInput,
   transaction
 ): Promise<Models.GamePieceAction> {
-  const targetGamePiece = await Models.GamePiece.findByPk(rangedAttackInput.targetGamePieceId, { transaction: transaction });
-
   // Validate attack data, raises exception if not valid
-  await resolveRangedAttackAction(
+  await validateRangedAttackAction(
```

Comment:
The separation of the `resolveRangedAttackAction` and `validateRangedAttackAction` methods makes this section a bit clearer.